fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+org = "ballerinax"
+name = "health.fhirr4"
+version = "${project.version}"
+distribution = "2201.4.1"
+authors = ["Ballerina"]
+keywords = ["Healthcare", "FHIR", "R4", "service"]
+repository="https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
+org="ballerinax"
+name="health.fhir.r4"
+version="1.0.0"
```

Comment:
shall we parameterize this as well? since each time new fhir r4 release we have to release the service as well, so we can miss to update here?

Reply:
done